chart/compass/values.yaml

```diff
@@ -136,6 +137,10 @@ global:
       port: 3002
       path: "/operation"
       lastOperationPath: "/last_operation"
+    info:
+      path: "/v1/info"
+      certIssuer: "C=BG, L=Local, O=CMP, OU=Local, CN=CMP"
```

Comment:
there is a generic mtlsClient config in 2057 after the latest comments there, you can pre-adapt it here so you dont depend on 2057